infra/gcb/build.py

```diff
@@ -199,8 +199,19 @@ def get_build_steps(project_yaml, dockerfile_path):
               # We also remove /work and /src to save disk space after a step.
               # Container Builder doesn't pass --rm to docker run yet.
               'rm -r /out && cd /src && cd {1} && mkdir -p {0} && compile && rm -rf /work && rm -rf /src'.format(out, workdir),
-              ],
-            },
+            ],
+           },
+          # test binaries
+          {'name': 'gcr.io/oss-fuzz-base/base_runner',
```

Comment:
Renaming done.

As discussed over chat, I'm making the script no-op unless `EXPERIMENTAL_BAD_BUILD_CHECK` is set to 1, and removing that build step untill I properly test it.